### PR TITLE
Fix documentation a bit

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+
 ## Upgrading
 
 * change the `promtail` upstream version in Chart dependencies (`helm/promtail/Chart.yaml`)
 * run `helm dependency update helm/promtail` to update the Chart.lock file
 * re-generate `helm/promtail/values.schema.json`:
   * `helm schema-gen helm/promtail/values.yaml > helm/promtail/values.schema.json` to re-generate the file.
   * `sed -i 's/"type": "null"/"type": ["string", "null"]/g' helm/promtail/values.schema.json` to accept strings for all null values.
-* update the link in the `Configuration` section of the Readmeto point to the new tag configuration.
+* update the link in the [`Configuration`](./README.md#configuration) section of the Readmeto point to the new tag configuration.
```
Comment (marieroque) [inline CONTRIBUTING.md:15]: Readmeto => Readme to
;-)

Comment (QuentinBisson) [reply CONTRIBUTING.md:15]: You know you can suggest changes ? 

image

image

Try it this is awesome :)

Comment (QuentinBisson) [inline CONTRIBUTING.md:15]:
```suggestion
* update the link in the [`Configuration`](./README.md#configuration) section of the Readme to point to the new tag configuration.
```

Comment (QuentinBisson) [reply CONTRIBUTING.md]: I applied it. Can you review one last time?

**Review by QuantumEnigmaa — APPROVED** [review]

**Pull request merged**